aur_test.go

```diff
@@ -46,48 +45,48 @@ func TestSearch(t *testing.T) {
 
 // TestSearchByName test searching for packages by package name
 func TestSearchByName(t *testing.T) {
-	rs, err := SearchByNameDesc("linux")
+	rs, err := SearchBy("linux", Name)
 	expectPackages(t, 100, rs, err)
 }
 
 // TestSearchByNameDesc test searching for packages package name and desc.
 func TestSearchByNameDesc(t *testing.T) {
-	rs, err := SearchByNameDesc("linux")
+	rs, err := SearchBy("linux", NameDesc)
 	expectPackages(t, 100, rs, err)
 }
 
 // TestSearchByMaintainer test searching for packages by maintainer
 func TestSearchByMaintainer(t *testing.T) {
-	rs, err := SearchByMaintainer("moscar")
+	rs, err := SearchBy("moscar", Maintainer)
 	expectPackages(t, 3, rs, err)
 }
 
 // TestOrphans test searching for orphans
-func TestOrphans(t *testing.T) {
-	rs, err := Orphans()
-	expectPackages(t, 500, rs, err)
-}
+//func TestOrphans(t *testing.T) {
+//	rs, err := Orphans()
+//	expectPackages(t, 500, rs, err)
+//}
```

Comment:
Could you add a comment here why it's disabled (basically what you had in the commit message).